function.php

```diff
@@ -360,7 +335,7 @@ function processPoly($point) {
 		if ($inside) {
 			debug("PolyHood gefunden...");
 			try {
-				$rs = db::getInstance()->prepare("SELECT ".hood_mysql_fields." FROM hoods WHERE id=:hoodid;");
+				$rs = db::getInstance()->prepare("SELECT ".hood_mysql_fields." FROM hoods WHERE id=:hoodid AND active=1;");
```

Comment:
Warum nehmen wir hier eigentlich nicht die function getHoodById?

Reply:
Gute Frage, war vorher schon so. Wäre aber eine Änderung mit commit wert.

Reply:
Genau, das hat mit dir jetzt nichts zu tun, war ne frage aus neugierde

Reply:
Ist in dem neuen PR schon mit drinnen